.github/workflows/release.yml

```
@@ -250,7 +258,7 @@ jobs:
       - name: Install Rust Toolchain
         uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
         with:
-          toolchain: stable
+          toolchain: nightly-2023-12-28
```

Using `RUSTC_BOOTSTRAP=1` below should allow using `stable` here.
```suggestion
          toolchain: stable
```